src/js/components/explorer/detail/visualization/treemap/ExplorerTreemap.jsx

```diff
@@ -46,12 +46,12 @@ export default class ExplorerTreemap extends React.Component {
         this.buildVirtualChart(this.props);
     }
 
-    componentWillReceiveProps(nextProps) {
-        if (nextProps.data !== this.props.data) {
-            this.buildVirtualChart(nextProps);
+    componentDidUpdate(prevProps) {
+        if (prevProps.data !== this.props.data) {
```

Comment:
`isEqual` is needed here as `data` is an object and the deep comparison operator won't be reliable.